blueocean-git-pipeline/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>blueocean-parent</artifactId>
+    <groupId>io.jenkins.blueocean</groupId>
+    <version>1.0.0-b15-SNAPSHOT</version>
+  </parent>
+
+  <name>Module :: BlueOcean :: Git Pipeline plugin</name>
```

Comment:
Could this be renamed to follow the same naming standaard that others now use? 

Something like Git Pipeline for blueocean plugin